src/Easydropdown/Easydropdown.test.ts

```diff
@@ -152,4 +152,30 @@ describe('Easydropdown', () => {
             assert.equal(edd.dom.item.length, 7);
         });
     });
+
+    describe('.validate()', () => { 
+
+        it('validates a required select', () => {
+            const select = createSelect('required');
+            const edd = new Easydropdown(select, {});
+
+            // @ts-ignore
```

Comment:
don't think you need these @ts-ignore - in the other tests, they were needed so we can test private methods, but validate is part of the public API so should be fine.